fe1-web/src/features/social/navigation/SocialMediaNavigation.tsx

```diff
@@ -72,11 +72,15 @@ const SocialMediaNavigation = () => {
       /* noop */
     });
 
+  const contextValue = useMemo(
+    () => ({
+      currentUserPopTokenPublicKey,
+    }),
+    [currentUserPopTokenPublicKey],
+  );
```

Comment:
isn't this a bit of a weird change in this PR ? What is it for?

Reply:
True! I think we should set the linter to be more aggressive in these cases, for some reason it was reported to me once but now not anymore. I assumed it was because of a eslint update but it seems it was something else..

Anyway, the purpose is to prevent unnecessary rerenders, react only performs shallow equality checks and before the value passed to the context always was a different object which would trigger a re-render in all components making use of the context. I will add a comment to make this clear.

Reply:
I guess it might be a good idea to find the corresponding linter rule and enable it